Commands/Public/twitter.js

```diff
@@ -0,0 +1,70 @@
+const getRSS = require("../../Modules/Utils/RSS.js");
+const PaginatedEmbed = require("../../Modules/MessageUtils/PaginatedEmbed");
+const moment = require("moment");
+
+module.exports = async ({ Constants: { Text, Colors, APIs } }, { serverDocument }, msg, commandData) => {
+	let { suffix } = msg;
+	if (suffix && suffix.startsWith("@")) {
+		suffix = suffix.slice(1);
+	}
+
+	if (suffix) {
+		let query = suffix.substring(0, suffix.lastIndexOf(" "));
+		let num = suffix.substring(suffix.lastIndexOf(" ") + 1);
+
+		if (!query || isNaN(num)) {
+			query = suffix;
+			num = serverDocument.config.command_fetch_properties.default_count;
+		}
+		if (num > serverDocument.config.command_fetch_properties.max_count) {
+			num = serverDocument.config.command_fetch_properties.max_count;
+		} else if (num < 1) {
+			num = serverDocument.config.command_fetch_properties.default_count;
+		} else {
+			num = parseInt(num);
+		}
+
+		const m = await msg.channel.send(Text.THIRD_PARTY_FETCH("The birbs are hard at work 🐦"));
```

Comment:
Or something more fun than what it was before

```suggestion
		const m = await msg.channel.send(Text.THIRD_PARTY_FETCH("The birbs are searching~ 🐦"));
```

Reply:
Don't you ever fucking call me out for a lack of fun.